README.md

```diff
@@ -48,3 +48,13 @@ For production build run:
     $ gulp build
 
 This will compile `assistant.user.js` file with minification and uglify.
+
+## Mobile build
+
+Mobile build is [here](https://AdguardTeam.github.io/AdguardAssistant/mobile/assistant.user.js)
+
+In the mobile assistant, the rules are specified via a function ADG_addRule:
```

Comment:
Mobile build should not be a userscript, just a script that you can load & run.

Signature was described in the issue